app/src/org/commcare/recovery/measures/ScanCczTask.java

```diff
@@ -94,6 +96,7 @@ private void locateAppCcz(File[] files, int depth) {
         if (!StringUtils.isEmpty(lastKnownCczLocation)) {
             filePathsToScan.add(new File(lastKnownCczLocation));
         }
+        Logger.log(LogTypes.SOFT_ASSERT, "Access outside scoped storage in scanning cczs.");
```

Comment:
We can remove this soft assert as well as this path will definitely get hit for someone using this functionality,